### ISSUE TITLE
Packages should be installed concurrently 

### ISSUE THREAD

**Issue by paulcalabro** [wazuh/wazuh-ansible#109]
When installing packages, it's encouraged to provide a _list_ of packages to install, as opposed to _iterating over a list one by one_. Most modern package managers can install multiple packages concurrently. When you using a loop, you're doing one transaction at a time, consecutively, so it's less performant and it generates warnings such as:

```
[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply multiple items and specifying 
`name: {{ item }}`, please use `name: ['apt-transport-https', 'ca-certificates']` and remove the loop. This feature will be removed in version 2.11. Deprecation 
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

**Comment by manuasir** [comment]
Hello @paulcalabro ,

We opened a new PR with the suggested changes. Much appreciated.
Feel free to review the new code or open a new issue whenever you may need it.
Thanks for your kind collaboration.

Cheers

**Comment by manuasir** [comment]
Merged #110

**Comment by LuisGi93** [comment]
Hi @paulcalabro ,
some research were done related with this issue and  modifications were made at https://github.com/wazuh/wazuh-ansible/pull/110 to fix this issue. Now a list of packages is used when using the apt module.

Regards.
